src/test/java/com/github/dockerjava/core/command/BuildImageCmdImplTest.java

```diff
@@ -274,6 +274,28 @@ public void labels() throws Exception {
         assertThat(inspectImageResponse.getConfig().getLabels().get("test"), equalTo("abc"));
     }
 
+    @Test
+    public void multipleTags() throws Exception {
+        if (!getVersion(dockerClient).isGreaterOrEqual(RemoteApiVersion.VERSION_1_21)) {
+            throw new SkipException("API version should be >= 1.21");
+        }
+
+        File baseDir = fileFromBuildTestResource("labels");
+
+        String imageId = dockerClient.buildImageCmd(baseDir).withNoCache(true)
+                .withTag("docker-java-test:tag1").withTag("docker-java-test:tag2")
```

Comment:
imho that looks not obvious. If user wants only one tag it should call `.withTag` if multiple then `.withTags(Set<String>)` Maybe keep `getTag()` and add `getTags()` and prefer getTags with fallback to getTag()?